### PR TITLE
Fix headers in documentation

### PR DESCRIPTION
**Pull request by thaJeztah**
These headers were rendered as body text because there was no space, but a "tab" after the `#`.

Took me a few minutes to find out why they didn't render as header, haha

### CODE REVIEW

File: docs/nginx.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ server {
   ssl_certificate /etc/nginx/conf.d/domain.crt;
   ssl_certificate_key /etc/nginx/conf.d/domain.key;
 
-  # Recommendations from https://raymii.org/s/tutorials/Strong_SSL_Security_On_nginx.html
+  # Recommendations from https://raymii.org/s/tutorials/Strong_SSL_Security_On_nginx.html
```
Comment (thaJeztah) [inline docs/nginx.md]: Yeah, not related, but replaced that one as well :)

**Comment by RichardScothern** [comment]
I can't test with the real documentation, but the change sounds OK and the github markdown renders ok.  LGTM

**Comment by thaJeztah** [comment]
@RichardScothern oh, you can; try running `make docs` in the docs directory and be amazed!

**Comment by RichardScothern** [comment]
I am amazed!  LGTM.

**Pull request merged**